packages/runtime/package.json

```diff
@@ -118,15 +118,11 @@
     "@emotion/utils": "^1.0.0",
     "@makeswift/next-plugin": "0.2.8",
     "@popmotion/popcorn": "^0.4.4",
-    "@types/cookie": "^0.5.1",
-    "@types/http-proxy": "^1.17.9",
     "@types/is-hotkey": "^0.1.7",
-    "@types/set-cookie-parser": "^2.4.2",
     "@types/use-sync-external-store": "^0.0.3",
     "@types/uuid": "^9.0.1",
     "@use-gesture/react": "^10.2.24",
     "color": "^3.1.3",
-    "cookie": "^0.5.0",
```

Comment:
I was also able to remove this and `set-cookie-parser` :feelsgood:

Reply:
Nice!